src/layouts/layout.astro

```diff
@@ -112,14 +112,14 @@ import package_ from "../../package.json";
 			<ul>
 				<li>
 					Mastodon (<span data-translate="incl">incl.</span>&#x0020;<span
-						data-enumerate="Hometown,Fedibird,GlitchCafé"
-						>Hometown, Fedibird, GlitchCafé</span
+						data-enumerate="Hometown,Fedibird,GlitchCafé,Chuckya"
+						>Hometown, Fedibird, GlitchCafé, Chuckya</span
 					>)
 				</li>
 				<li>
 					Misskey (<span data-translate="incl">incl.</span>&#x0020;<span
-						data-enumerate="Firefish,Calckey,FoundKey,Meisskey"
-						>Firefish, Calckey, FoundKey, Meisskey</span
```

Comment:
**issue:** Meisskey is **not a typo**, but a fork of Misskey. It might be unpopular, but we support it, so let's keep it in the list

Reply:
ah ok